### PR TITLE
Onboarding page

### PR DESCRIPTION
**Pull request by Mickaellah**
Hello team,

Team Dong has just finished the onboarding page which is the registration page.

Please have a look at it when you are available.

Branch: https://github.com/onja-org/vamalaka-dong/tree/onboarding-page

Thanks in advance.

### CODE REVIEW

File: src/components/RegistrationOnboarding/RegistrationOnboarding.tsx [hunk 1]
```diff
@@ -151,132 +153,107 @@ const Container = styled("div")`
...
+  cursor: pointer;
+  position: absolute;
+  top: 33px;
+  right: 21px;
+  font-family: Futura Std;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 14px;
+  display: flex;
```
Comment (Tanosimboangy) [inline src/components/RegistrationOnboarding/RegistrationOnboarding.tsx]: Should be removed since you are using position absolute.

**Review by Tanosimboangy — APPROVED** [review]

**Review by Ny-Rianala — APPROVED** [review]

File: src/components/RoleSelectOnboard/RoleSelectOnboard.stories.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Story, Meta } from '@storybook/react'
-import { RoleSelectOnboard } from './RoleSelectOnboard'
-import { Props } from '../PageFooter/PageFooter'
+import { RoleSelectOnboard, RoleSelectOnboardProps } from './RoleSelectOnboard'
+// import { Props } from '../PageFooter/PageFooter'
```
Comment (andrzej-onja) [inline src/components/RoleSelectOnboard/RoleSelectOnboard.stories.tsx]: remove unused code

File: src/pages/OnboardingPage/OnboardingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    isChecked: boolean,
+    isPrimary: boolean,
+}
+
+export const OnboardingPage: React.FC<OnboardingPageProps> = ({
+    footerLinks, 
+    isChecked,
+    isPrimary,
+}) => {
+    const [isClicked, setIsClicked] = useState(false);
```
Comment (andrzej-onja) [inline src/pages/OnboardingPage/OnboardingPage.tsx]: lets rename it to be less generic (selectedRole) and accept selected role in handle click as a string.

Comment (andrzej-onja) [reply src/pages/OnboardingPage/OnboardingPage.tsx]: Role value will have to be sent to the backend in registration request

File: src/components/RegistrationOnboarding/RegistrationOnboarding.tsx [hunk 4]
```diff
@@ -143,159 +147,118 @@ const Container = styled('div')`
...
+                        Register your account
+                    </h3>
+                    <div>
+                        <Back label="Back" href="/" />
+                        <div>
+                            <p>Step 01/02</p>
+                            <p>Personal info.</p>
+                        </div>
+                    </div>
+                </header>
```
Comment (andrzej-onja) [inline src/components/RegistrationOnboarding/RegistrationOnboarding.tsx:223]: extract header as a separate component 
Reuse on RoleSelection, registration form and registration success

Comment (andrzej-onja) [reply src/components/RegistrationOnboarding/RegistrationOnboarding.tsx:223]: or add it somehow if its possible to OnboardingPage

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**